examples/misc/agent_services.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+'''
+Demonstrate the features of Agent Services.
+'''
+
+import os
+import sys
+
+import radical.utils as ru
+import radical.pilot as rp
+import platform
+import socket
+import re
+import uuid
+import logging
+
+
+logger = ru.Logger('agent_services')
+
+
+
+def get_system_info():
+    try:
+        info = {}
```

Comment:
please use 
```py
info = dict()
```
here (and similar in other places).  Python's short nomenclature is harder to parse (for humans) and is also not consistent.  For example:
```py
>>> type([])
<class 'list'>
>>> type([2])
<class 'list'>

>>> type(())
<class 'tuple'>
>>> type((2))
<class 'int'>
```

Reply:
Yes agree. But now file removed.